requirements/prod.in

```diff
@@ -2,15 +2,15 @@ APScheduler==3.11.0
 babis==0.2.4
 basket-client==1.2.0
 bleach[css]==6.1.0
-boto3==1.35.99
+boto3==1.36.26
 chardet==5.2.0
 commonware==0.6.0
-contentful==2.2.0
+contentful==2.3.2
 contextlib2==21.6.0
 dirsync==2.2.5
 dj-database-url==2.3.0
 django-allow-cidr==0.7.1
-django-cors-headers==4.6.0
+django-cors-headers==4.7.0
 django-crum==0.7.9
 django-csp==4.0b2
```

Comment:
Would you mind manually bumping this to `django-csp==4.0b4`? I tested locally and it passes.